release/pkg/manager/calico/manager.go

```diff
@@ -411,28 +412,35 @@ func (r *CalicoManager) TagRelease(ver string) error {
 	logrus.WithFields(logrus.Fields{"branch": branch, "version": ver}).Infof("Creating Calico release from branch")
 	_, err := r.git("tag", ver)
 	if err != nil {
-		return fmt.Errorf("Failed to tag release: %s", err)
+		return fmt.Errorf("failed to tag release: %s", err)
+	}
+	return nil
+}
+
+func (r *CalicoManager) modifyHelmValues() error {
```

Comment:
This change seems fine to me, I just don't quite understand why we're making it - is there a motivation behind this?